src/platform/web/scrollcomponent/ScrollEventNormalizer.ts

```diff
@@ -36,10 +36,10 @@ export class ScrollEventNormalizer {
             nativeEvent: {
                 contentOffset: {
                     get x(): number {
-                        return window.scrollX;
+                        return window.scrollX === undefined ? window.pageXOffset : window.scrollX;
```

Comment:
Hmm, good question. I think on supported browsers the answer would be: "yes"

At the moment my personal aim for browser support is modern Chrome, Safari, Firefox and Edge, as well as IE11 which is still supported by Microsoft. Of those, the only browser that still uses the old implementation is IE11 where `pageXOffset` will always be available.

I found the following on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollX#Notes):

> The pageXOffset property is an alias for the scrollX property:

```js
window.pageXOffset == window.scrollX; // always true
```

> For cross-browser compatibility, use window.pageXOffset instead of window.scrollX. Additionally, older versions of Internet Explorer (< 9) do not support either property and must be worked around by checking other non-standard properties. A fully compatible example:

```js
var x = (window.pageXOffset !== undefined)
  ? window.pageXOffset
  : (document.documentElement || document.body.parentNode || document.body).scrollLeft;
```

I'm not sure we want to go that far though. It depends on your idea for browser support in RLV. Either way, at the moment RLV is just broken in IE11 and the proposed implementation will at least fix that, even if it stays broken in other edge case browers.